data/json/mutations/mutation_eocs.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_BIOLUM3_activated",
+    "condition": { "compare_int": [ { "u_val": "stamina" }, ">=", { "const": 3000 } ] },
+    "effect": [
+      { "u_assign_activity": "ACT_GENERIC_EOC", "duration": "3 seconds" },
+      { "arithmetic": [ { "u_val": "stamina" }, "-=", { "const": 700 } ] },
```

Comment:
So if I understand this, the stamina cost is now 700 and it's paid during the EOC activation and since it's paid by the time the spell is cast, the spell costs 0?

Reply:
Yes, that's correct.  After the changes to maximum stamina, 350 stamina is (proportionally) much less to a healthy player, from my understanding, so I upped the cost.